blocks/edit/prose/index.js

```diff
@@ -116,11 +117,12 @@ function setPreviewBody(daPreview, proseEl) {
   daPreview.body = body;
 }
 
-function pollForUpdates() {
-  const daContent = document.querySelector('da-content');
+export function pollForUpdates(doc = document, win = window) {
```

Comment:
Just a suggestion - I prefer using an IIFE to contain the `pollerSetup` state outside global scope:

```
const pollForUpdates = (() => {
  let pollerSetUp = false;

  return (doc = document, win = window) => {
    if (pollerSetUp) return;
    const daContent = doc.querySelector('da-content');
    const daPreview = daContent.shadowRoot.querySelector('da-preview');
    if (!win.view) return;
    const proseEl = win.view.root.querySelector('.ProseMirror');
    if (!daPreview) return;

    setInterval(() => {
      if (sendUpdates) {
        if (hasChanged > 0) {
          hasChanged = 0;
          return;
        }
        setPreviewBody(daPreview, proseEl);
        sendUpdates = false;
      }
    }, 500);
    pollerSetUp = true;
  };
})();
```